### DISCUSSION TITLE
File over Ethercat for firmware updates

### DISCUSSION THREAD

**Discussion by djsjoerdsma** [ethercrab-rs/ethercrab#235] [Ideas]
Hi all! Just checkout out this awesome growing project. I was curious if there was plans to add FoE capability? I am assuming it would be similar to the CoE upload/download functionality?

**Comment by jamwaffles** [comment]
Hey, I'd like to add it as a feature for sure, but I don't have a personal need for it so it's not in my current plans. I'd be happy to discuss sponsorship or other paid arrangements if it's a feature you need for an upcoming product though!
